File: Programmers/level1/K번째수.cc
#include <algorithm>
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {

	vector<int> answer;

	for (int i = 0; i < (int)commands.size(); ++i) {
			int start = commands[i][0] -1;
			int cut = commands[i][1];
			int find_num = commands[i][2]-1;

			vector<int> vec;
			for (int ii = start; ii < cut; ++ii) {
				vec.push_back(array[ii]);
			}
			sort(vec.begin(), vec.end());
			answer.push_back(vec[find_num]);
	}

	return answer;
}

File: Programmers/level1/K번째수.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

class Solution {
    public List<Integer> solution(int[] array, int[][] commands) {
        List<Integer> answer = new ArrayList<>();

        for (int[] cmd : commands) {
            int st = cmd[0] - 1;
            int ed = cmd[1] - 1;
            int target = cmd[2] -1;

            List<Integer> list = new ArrayList<>();
            for(int i = st; i <= ed; ++i){
                list.add(array[i]);
            }
            Collections.sort(list);
            answer.add(list.get(target));
        }

        return answer;
    }
}